src/ucp/rndv/proto_rndv.c

```diff
@@ -251,46 +234,19 @@ void ucp_proto_rndv_ctrl_config_str(size_t min_length, size_t max_length,
                                     const void *priv, ucs_string_buffer_t *strb)
 {
     const ucp_proto_rndv_ctrl_priv_t *rpriv = priv;
-    const ucp_proto_threshold_elem_t *thresh_elem;
-    size_t range_start, range_end;
-    const ucp_proto_t *proto;
     ucp_md_index_t md_index;
-    char str[64];
 
     /* Print message lane and memory domains list */
-    ucs_string_buffer_appendf(strb, "cln:%d md:", rpriv->lane);
+    ucs_string_buffer_appendf(strb, "ln:%d md:", rpriv->lane);
```

Comment:
`ln` -> `lane`?

Reply:
wanted to make it as short as possible, since nested rndv protocol description can be long